src/practitest_firecracker/practitest.clj

```diff
@@ -23,9 +27,11 @@
           "both `query-params` and `form-params` can't be specified")
   (loop [results []
          uri     uri
-         params  (cond-> {:basic-auth credentials
-                          :throw-exceptions false
-                          :as :json}
+         params  (cond-> {:basic-auth          credentials
+                          :throw-exceptions    false
+                          :as                  :json
+                          :source              "firecracker"
+                          :firecracker-version fc-version}
```

Comment:
Nope, you need to pass it to the server. This does not pass it to the server.

Reply:
Will change it to merge into query-params/report-params